src/main/java/io/jenkins/plugins/synopsys/security/scan/global/Utility.java

```diff
@@ -216,4 +217,34 @@ public static String jenkinsJobType(EnvVars envVars) {
             return "UnknownJobType";
         }
     }
+
+    public static String getDefaultSarifReportFilePath(boolean isBlackDuckScan, boolean isPolarisDuckScan) {
```

Comment:
Add unit tests for all the new methods (there are 4 including this) of this class